examples/applications/plot_out_of_core_classification.py

```diff
@@ -168,14 +167,14 @@ def progress(blocknum, bs, size):
             total_sz_mb = '%.2f MB' % (size / 1e6)
             current_sz_mb = '%.2f MB' % ((blocknum * bs) / 1e6)
             if _not_in_sphinx():
-                print('\rdownloaded %s / %s' % (current_sz_mb, total_sz_mb),
-                      end='')
+                sys.stdout.write(
```

Comment:
I did not remember right away why I did this. This is related to a sphinx-gallery bug, that `__future__` imports do not work across cells in a notebook-like example (i.e. that uses lines of hashes for separate cells). See https://github.com/sphinx-gallery/sphinx-gallery/pull/308 for more details.

Reply:
That's frustrating. I suppose that a run of the examples in Py2 serves as a regression test at least.

Reply:
Agreed, actually this is how I discovered the sphinx-gallery bug ...